x-pack/auditbeat/module/system/host/host.go

```diff
@@ -54,6 +54,19 @@ func (ms *MetricSet) Fetch(report mb.ReporterV2) {
 		return
 	}
 
+	users, err := GetUsers()
```

Comment:
You'll need a `GetUsers` stub with a build tag like `// +build windows !cgo` (meaning `windows OR (not CGO))`).